src/sass/main.scss

```diff
@@ -8,7 +8,10 @@ $DEFAULT_PADDING: 0.75rem;
         height: $ICON_SIZE;
         text-align: center;
         line-height: $ICON_SIZE;
-    }
+    };
+    & img {
+      vertical-align: text-bottom;
+    };    
```

Comment:
You might use `img` in your custom styling, but the default `toggle` does not contain `img`. So I wouldn't modify it here.